man/docker-run.1.md

```diff
@@ -190,8 +191,13 @@ the other shell to view a list of the running containers. You can reattach to a
 detached container with **docker attach**. If you choose to run a container in
 the detached mode, then you cannot use the **-rm** option.
 
-   When attached in the tty mode, you can detach from a running container without
-stopping the process by pressing the keys CTRL-P CTRL-Q.
+   When attached in the tty mode, you can detach from the container (and leave it
```

Comment:
Hum, I didn't copy it everywhere, true :stuck_out_tongue_closed_eyes:. I just updated this one as it was present :smile_cat: